content/developer/reference/backend/views.rst

```diff
@@ -976,6 +976,38 @@ system. Available semantic components are:
   ``attrs``
     same as for ``field`` component.
 
+  .. _reference/views/form/setting:
+``setting``
+  The ``setting`` tag is used to declare a setting, an entity visually separated on two panels
+  (left and right), and used to edit a given field. The first field in the setting is used as the
+  main field (optional). This field is placed on the left panel (if it's a boolean field) or on the
+  top of the right panel (otherwise). The field is also used to create the setting label if a
+  ``string`` is not defined. The ``setting`` tag can also contain more elements (e.g. html), all of
+  these elements are rendered in the right panel.
+
+  .. rst-class:: o-definition-list
+
+  ``string`` (optional)
+    The text used as label of the setting. If it's not defined, the first field is used as label.
+  ``title`` (optional)
+    The text used as tooltip.
+  ``help`` (optional)
+    The help/description of the setting. This text is displayed just below the setting label (with classname ``text-muted``).
+  ``company_dependent`` (optional)
+    If this attribute is set to "1" an icon is displayed next to the setting label to explicit
+    that this setting is company-specific.
+  ``documentation`` (optional)
+    If this attribute is set, an icon is added next to the setting label, this icon is a link to the documentation.
+    Note that you can use relative or absolute path. The relative path is relative to ``https://www.odoo.com/documentation/<server_version>``,
+    so it's not necessary to hard-code the server version on the arch anymore.
+
+  .. code-block:: xml
+
+    <setting string="this is bar">
+        <field name="bar"/>
+        ...More elements
+    </setting>
```

Comment:
Use the directive `example` to highlight examples and separate them from the reference text.
```suggestion
  .. example::
     .. code-block:: XML

       <setting string="this is bar">
           <field name="bar"/>
           ...More elements
       </setting>
```